test/core_tests.js

```diff
@@ -78,6 +79,11 @@
       expected = "ease-in-out";
       assert.strictEqual( actual, expected, "canvas.style.transitionTimingFunction initialized correctly" );
 
+      // Ensure we specify perspective independent of transform for IE 11
```

Comment:
Maybe if we change the message for something like `"root.style.perspective should be set explicitly for IE 11"` then we wouldn't need the comment and it would show the "why" when the test fails in the console.

The other tests are like "`n` initialized correctly" because we don't really had a reason for testing that attribute, it was just to have some blind coverage somewhere (which actually worked since when we added SauceLabs the tests failed in Safari xD).